pkg/mqtt/subscriber_manager.go

```diff
@@ -0,0 +1,106 @@
+package mqtt
+
+import (
+	"sync"
+)
+
+type OnSubscribeH func(topic string, id string)
+type OnUnsubscribeH func(topic string, id string)
+type OnConnectH func(id string)
+type OnDisconnectH func(id string)
+
+type subscriberManager struct {
```

Comment:
Add the description what the subscriptionManager does (take the one from the topic manager and extend it)